common/components/AddressFieldFactory/AddressFieldDropdown.scss

```diff
@@ -0,0 +1,42 @@
+@import 'common/sass/variables';
+@import 'common/sass/variables/colors';
+@import 'common/sass/variables/spacing';
+
+.AddressFieldDropdown {
+  position: absolute;
+  top: calc(100% - 16px);
+  border: 1px solid $gray-lighter;
+  border-top: none;
+  background: $component-active-color;
+  z-index: 1;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
```

Comment:
I'm actually the opposite, I think that dropdown style throughout the site should remain consistent. Maybe this would be a good time to change how we do dropdowns / input styles?

All the other dropdowns just follow the simple input style, but I'm definitely open to updating those. I'm also not opposed to the drop-shadow, but I do think dropdowns should match the input style.

Reply:
That may be the case, but the scope on this has already exploded. This would probably be better as a future story.